app/controllers/admin/organization/users_controller.rb

```diff
@@ -0,0 +1,29 @@
+class Admin::Organization::UsersController < ApplicationController
+
+  def index
+    @organization = Organization.find_by(slug: params[:organization_slug])
+    @admins = @organization.users
+    # require "pry"; binding.pry
+  end
+
+  def new
+    @organization = Organization.find_by(slug: params[:organization_slug])
+    @user = @organization.users.new
+  end
+
+  def create
+    organization = Organization.find_by(slug: params[:organization_slug])
+    role = Role.find_or_create_by(name: "org_admin")
+    user = User.find_by(username: params[:user][:username])
+    if user
+      user.roles << role
+      org_users = organization.users
+      org_users << user
+      # flash[:success] = "You've added #{user.username} as an admin"
```

Comment:
@jmejia
We figured it out.. we were validating presence of password which was redundant and causing a password to be required for updating.